accounts/urls.py

```diff
@@ -6,9 +7,9 @@
 
 urlpatterns = [
     path("signup/", views.SignUpView.as_view(), name="signup"),
-    # path('login/', auth_views.LoginView.as_view(), name='login'),
-    # path('logout/', auth_views.LogoutView.as_view(), name='logout'),
-    # path('<str:username>/', views.UserProfileView.as_view(), name='user_profile'),
+    path("login/", views.LoginView.as_view(), name="login"),
+    path("logout/", auth_views.LogoutView.as_view(), name="logout"),
+    path("<slug:username>/", views.UserProfileView.as_view(), name="user_profile"),

```

Comment:
```suggestion
    path("<str:username>/", views.UserProfileView.as_view(), name="user_profile"),
```
[django.contrib.auth.models.User](https://docs.djangoproject.com/ja/4.1/ref/contrib/auth/#django.contrib.auth.models.User.username) の `username` は英数字の他に `_`, `@`, `+`, `.`, `-` が使えますが，[slug](https://docs.djangoproject.com/en/4.1/ref/validators/#validate-slug) ではそれらをカバーしきれないので普通に `str` で良いです．